2020/02/04 01:53:09 [error] 6792#6792: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2020/02/04 01:53:23 [error] 6792#6792: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2020/02/04 01:53:30 [error] 6792#6792: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2020/02/04 01:53:42 [error] 6792#6792: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2020/02/04 01:56:00 [crit] 7479#7479: *1 connect() to unix:/var/run/php7.2-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php7.2-fpm.sock:", host: "yiitest.loc"
2020/02/04 02:00:04 [error] 7717#7717: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "yiitest.loc"
2020/02/04 02:04:03 [crit] 7919#7919: *4 connect() to unix:/var/run/php7.2-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: yiitest.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php7.2-fpm.sock:", host: "yiitest.loc"
